recipes/pyarrow/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.pre" %}
+{% set commit = "772800ace3706b8f169803b3ef8932e04ec54619" %}
+
+package:
+  name: pyarrow
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.tar.gz
+  url: https://github.com/apache/arrow/archive/{{ commit }}.tar.gz
+  sha256: 01ff68b9ab2785cc2a465fedc62536faab4ff461b6d42b9b00df15bb2e41dfa5
+
+build:
+  number: 0
+  rpaths:
+    - lib                                                        # [unix]
+    - lib/python{{environ.get('PY_VER')}}/site-packages/pyarrow  # [unix]
```

Comment:
The linter is failing here. Would `{{ environ['PY_VER'] + '*' }}` work instead? That is known to pass.